packages/ListBox/src/components/Option/Option.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import PropTypes from "prop-types";
+import useListBox from "../../useListBox";
+import { getA11yAttributesForOption } from "../../helpers/DOMAttributes";
+import { isOptionVisible, isOptionSelected, handleClickOption } from "../Options/helpers/options";
+import { OptionStyled } from "./Option.styles";
+
+const propTypes = {
+  /** String, number or JSX content */
+  children: PropTypes.oneOfType([PropTypes.node, PropTypes.func]).isRequired,
+
+  /** Describe if the option is selected or not */
+  isSelected: PropTypes.bool,
+
+  /** Describe if the option is enable or not */
```

Comment:
Not sure how much value some of these comments are adding.
If your response is `They are for auto-documentation` then my response to that is `then how come more components like the Filter and Footer don't have them?` :)